app/containers/LandingPage/LandingPage.tsx

```diff
@@ -43,36 +42,45 @@ interface LandingPageState {
 }
 
 class LandingPage extends React.PureComponent<LandingPageProps, LandingPageState> {
-  render() {
-    return <ApolloConsumer>
-      {client => this.renderMain(client)}
-    </ApolloConsumer>;
-  }
-  onSubmit = async (e,client:ApolloClient<any>) => {
+  state: LandingPageState = {
+    searchTerm: '',
+  };
+  onSubmit = async (e, client:ApolloClient<any>) => {
```

Comment:
This might be bugged with regards to the ApolloConsumer part, but I think it works regardless. This is to make it so you can simply hit enter on the landing page search box and it will take you to an empty search. Mostly for testing purposes, but it should be useful for the end user too I think.